maas/resource_maas_instance_test.go

```diff
@@ -110,6 +151,46 @@ func testAccMAASInstanceCheckMachineLogsForDestroy(hostname string, erase bool)
 	}
 }
 
+func testAccMAASInstanceCheckReleaseScriptsRanOnDestroy(hostname, scriptName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		conn := testutils.TestAccProvider.Meta().(*maas.ClientConfig).Client
+		releaseScriptRan := false
+
+		for _, rs := range s.RootModule().Resources {
+			if rs.Type != "maas_vm_host_machine" {
+				continue
+			}
+
+			if rs.Primary.Attributes["hostname"] != hostname {
+				continue
+			}
+
+			params := entity.NodeResultParams{
+				Type: 3,
```

Comment:
I think this can also work and it is more intuitive:

```suggestion
				Type: entity.RELEASE,
```